menelaos-front/.editorconfig

```diff
@@ -0,0 +1,13 @@
+# Editor configuration, see https://editorconfig.org
```

Comment:
Should be removed

Reply:
By generating .gitignore file, Angular says it shouldn't